launcher/agent/agent.go

```diff
@@ -51,7 +51,9 @@ type AttestationAgent interface {
 
 type attestRoot interface {
 	// Extend measures the cel content into a measurement register and appends to the CEL.
-	Extend(cel.Content, *cel.CEL) error
+	Extend(cel.Content) error
+	// Get Canonical Event Log, The CEL should be consistent
```

Comment:
", The CEL should be consistent"?

Maybe "GetCEL fetches the CEL with events corresponding to the sequence of `Extend`ed measurements attestation root."

Reply:
yes, updated